pkgs/development/python-modules/python-nvd3/default.nix

```
@@ -1,20 +1,18 @@
-{ lib, buildPythonPackage, fetchPypi, python-slugify, jinja2, setuptools }:
+{ lib, buildPythonPackage, fetchFromGitHub, python-slugify, jinja2, setuptools, pytest }:
 
 buildPythonPackage rec {
   pname = "python-nvd3";
   version = "0.15.0";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "05bph75x6ss0nbavwylsbam28h4bvh88nfjgmas5bwhfgvs5zmzv";
+  src = fetchFromGitHub {
+    owner = "areski";
+    repo = "python-nvd3";
+    rev = "dc8e772597ed72f413b229856fc9a3318e57fcfc";
```

```suggestion
    # https://github.com/areski/python-nvd3/issues/172
    rev = "dc8e772597ed72f413b229856fc9a3318e57fcfc";
```
how do you know that this rev is 0.15.0?

---

I compared with the tar.gz on PyPi - python source files are identical, only the build and test supporting files differ.